build/tasks/ProjectModel/PatchPolicy.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Frameworks;
+
+namespace RepoTasks.ProjectModel
+{
+    [Flags]
+    internal enum PatchPolicy
+    {
+        /// <summary>
+        ///     Only produce new package versions if there were changes to product code.
+        /// </summary>
+        ProductChangesOnly = 1 << 0,
```

Comment:
Sorry if this is obvious but I didn't see how this was used. Where are you checking to ensure that if this flag is set, we will check whether the repo contains new changes to the product code and therefore requires an update?

Reply:
Not in this update, but hopefully soon when we finally address https://github.com/aspnet/Coherence-Signed/issues/858